drag_and_drop_v2/drag_and_drop_v2.py

```diff
@@ -444,10 +509,7 @@ def _get_item_state(self):
         state = {}
 
         for item_id, item in self.item_state.iteritems():
-            if isinstance(item, dict):
-                state[item_id] = item
-            else:
-                state[item_id] = {'top': item[0], 'left': item[1]}
+            state[item_id] = StateMigration.apply_item_state_migrations(item)
```

Comment:
@e-kolpakov Thanks for factoring the migration logic out to utils.py! Makes the code much more readable +1